### PR TITLE
Replace `<Form> <FormControl> <FormGroup> <HelpBlock>` with HTML equi…

### PR DESCRIPTION
**Pull request by rhamilto**
…valents in storage class form

### CODE REVIEW

**Comment by rhamilto** [comment]
/hold cancel

@spadgett PTAL

File: frontend/public/components/storage-class-form.tsx [hunk 1]
```diff
@@ -834,6 +833,10 @@ export class StorageClassForm_ extends React.Component<
     }
 
     const dynamicContent = _.map(parameters, (parameter, key) => {
+      const paramId = `storage-class-provisioner-${_.get(parameter, 'name', key).replace(
```
Comment (spadgett) [inline frontend/public/components/storage-class-form.tsx]: Maybe `_.kebabCase`?

```suggestion
      const paramID = `storage-class-provisioner-${_.kebabCase(_.get(parameter, 'name', key))}`;
```

Comment (rhamilto) [reply frontend/public/components/storage-class-form.tsx]: Neat!  Updated.

**Pull request merged**